src/current/_includes/v25.1/known-limitations/alter-column-limitations.md

```diff
@@ -9,4 +11,4 @@ You cannot alter the data type of a column if:
 
 {{site.data.alerts.callout_info}}
 Most `ALTER COLUMN TYPE` changes are finalized asynchronously. Schema changes on the table with the altered column may be restricted, and writes to the altered column may be rejected until the schema change is finalized.
```

Comment:
This behavior no longer occurs and was an artifact of the legacy schema changer. While changes are still processed in a background job and may be asynchronous for the user, we no longer restrict or reject modifications on the table.

Reply:
Removed!